pyaedt/modeler/Primitives3D.py

```diff
@@ -1559,3 +1560,997 @@ def add_environment(
         environment.insert(self._app)
         self.multiparts.append(environment)
         return environment
+
+    @aedt_exception_handler
+    def create_choke(self, json_file):
+        """Create a chock from json setting file.
+
+        Parameters
+        ----------
+        json_file : str
+            Full path of the json file return for the function check_choke_values.
+
+        Returns
+        -------
+        List of
+            bool
+                ``True`` when successful, ``False`` when failed.
+            :class:`pyaedt.modeler.Object3d.Object3d`
+                3D object core.
+            list of
+                :class:`pyaedt.modeler.Object3d.Object3d`
+                    3D object winding.
+                list
+                    list of point coordinates of the winding.
+            for each winding.
+
+        Examples
+        --------
+        Json file has to be like the following example.
+
+        >>> from pyaedt import Hfss
+        >>> hfss = Hfss()
+        >>> dictionary_values = hfss.modeler.check_choke_values("C:/Example/Of/Path/myJsonFile.json")
+        >>> mychoke = hfss.modeler.create_choke("C:/Example/Of/Path/myJsonFile_Corrected.json")
+        """
+        json_file = open(json_file, "r")
+        values = json.load(json_file)
+        self.logger.info("CHOKE INFO" + str(values))
+
+        sr = 1.1
+        n_seg = 0
```

Comment:
I would just rename those variables so that, just reading their name we can understand quickly what is their purpose.

Reply:
Yes sure, n_seg will become segment_number and sr, security factor